tests/functions/test_map_to_s3.py

```diff
@@ -8,6 +8,24 @@
 CONF_PATH = "cdn_lambda/functions/map_to_s3/map_to_s3.json"
 
 
+@pytest.mark.parametrize(
+    "uri",
+    [
+        "www.example.com/content/origin/rpms/repo/ver/filename",
+        "www.example.com/origin/rpm/repo/ver/filename",
+        "www.example.com/content/origin/rpm/repo/ver/filename",
```

Comment:
Relating to that point about "key in" vs "startswith", it would be good to also add a case where the alias appears somewhere other than the beginning of a URL here.

Reply:
Done.